drivers/shared/src/make_dmy.c

```diff
@@ -29,10 +29,6 @@
 #include <vic_run.h>
 #include <vic_driver_shared.h>
 
-#ifndef _LEAPYR
-#define LEAPYR(y) (!((y) % 400) || (!((y) % 4) && ((y) % 100)))
-#endif
-
 /******************************************************************************
```

Comment:
There is now a calendar specific function in vic_time.c `leap_year(year, calendar);`.